src/main/java/roomescape/domain_entity/Reservation.java

```diff
@@ -0,0 +1,44 @@
+package roomescape.domain_entity;
```

Comment:
```suggestion
package roomescape.domain;
```

domain과 entity를 굳이 패키지로 묶어볼 필요는 없는 것 같아요. domain, entity에 대한 개념이 물론 분리되어있는 것은 맞으나 현재 미션 내에서는 domain에 더 집중해보고 entity는 그것에 포함된다라고 보면 좋겠어요.

다른 부분들에 집중할 내용들도 충분히 많아보이네요.